### DISCUSSION TITLE
Asset store availability?

### DISCUSSION THREAD

**Discussion by rubmon64** [Orange-Panda/Rewired-Glyphs#6] [General]
Thank you for this amazing asset! Please consider releasing on the Unity asset store, I'm pretty sure this would be a hit as it's really needed and I don't see anything else like it there.
Cheers!

**Comment by Orange-Panda** [comment]
This will be considered but it will not be a high priority in the near future unless there is a significant demand for it

**Reply by LyingTurkey** [reply]
please do. would be much more convenient considering i seem to be having trouble importing into unity 6000.0.42f1

**Reply by Orange-Panda** [reply]
Noted, what issues are you encountering with the import process in Unity 6000.0.42f1?

**Reply by LyingTurkey** [reply]
I'm a bit late replying to this thread, but i figured the issue out the next day! It was user error on my end, the asset was imported properly! Aside from that, i really think it would be pretty convenient for users to have the ability to update directly from the asset store if you ever added bugfixes, extra components, etc. Overall this is a great asset, and you're a lifesaver!!
